models/intermediate/int_transform_contract_code.sql

```diff
@@ -0,0 +1,175 @@
+{{
+    config(
+        materialized = 'table',
+        unique_key = ['ledger_key_hash', 'closed_at'],
+        cluster_by = ["ledger_key_hash", "closed_at", "row_hash"],
+        tags = ["soroban_analytics", "intermediate", "daily"]
```

Comment:
What are the `intermediate` and `daily` tags for?

Reply:
Added these tags to segregate Models by processing layers(intermediate) and frequency as we have airflow jobs scheduled at varying frequency. Also, since all these Type 2 Models will go into their own DAG, added additional tags to control run behavior in case of issues with Type 2 data

Reply:
If that's the case I think the names need to be more descriptive. `intermediate` and `daily` without any qualifiers would be hard to understand from a user perspective

Reply:
sure. we can edit them as needed but my intent was i have 3 Models in this set that are intermediate.
If I ran to run them all i can do something like 
`dbt run --select tag:soroban_analytics tag:intermediate`